package.json

```diff
@@ -25,6 +28,8 @@
     "husky": "4.2.5",
     "lint-staged": "10.2.11",
     "markdown-it-anchor": "6.0.0",
+    "netlify-cli": "2.65.6",
+    "netlify-lambda": "2.0.1",
```

Comment:
@will0101 I just read through the docs for `netlify-lambda` and I'm pretty sure we don't need it.

> Use `netlify-lambda` only if you need a build step for your functions.

From https://github.com/netlify/netlify-lambda#when-to-use-netlify-dev-or-netlify-lambda-or-both

We don't need a build step for our functions, so there's no reason that I can see for including the dependency or the `start:lambda` script. Let me know if I'm missing something there.

Reply:
Fixed! We don't need a build step.